lib/matplotlib/pyplot.py

```diff
@@ -333,10 +324,9 @@ def gci():
 ## Any Artist ##
 
 
-# (getp is simply imported)
-@docstring.copy(_setp)
-def setp(obj, *args, **kwargs):
-    return _setp(obj, *args, **kwargs)
+get = matplotlib.artist.get
```

Comment:
Can we add a short comment like the one above.  These don't look like API to me when I look at them, they look like local aliases to the methods that someone didn't import right...